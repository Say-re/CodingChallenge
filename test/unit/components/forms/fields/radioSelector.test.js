import React from 'react';
import { shallow } from 'enzyme';
import RadioSelectorComponent from '../../../../../src/components/forms/fields/radioSelector';

describe('Radio Selection Component Tests', () => {
  test('RadioSelectorComponent Match Snapshot Test', async () => {
    const wrapper = shallow(<RadioSelectorComponent />)
    await expect(wrapper).toMatchSnapshot();
  })
})