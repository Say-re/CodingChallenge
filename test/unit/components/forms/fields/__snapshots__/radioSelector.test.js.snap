// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio Selection Component Tests RadioSelectorComponent Match Snapshot Test 1`] = `
<div
  className="blockFormFields"
>
  <p>
    Please choose the number of years you have been developing:
  </p>
  <div
    className="radio"
  >
    <input
      id="one"
      name="experienceLevel"
      onChange={[Function]}
      type="radio"
      value="Less than one year"
    />
    <label
      for="one"
    >
      Less than one year
    </label>
  </div>
  <div
    className="radio"
  >
    <input
      id="two"
      name="experienceLevel"
      onChange={[Function]}
      type="radio"
      value="More than 1 year but less than 3"
    />
    <label
      for="two"
    >
      More than 1 year but less than 3
    </label>
  </div>
  <div
    className="radio"
  >
    <input
      id="three"
      name="experienceLevel"
      onChange={[Function]}
      type="radio"
      value="More than 3 years but less than 5"
    />
    <label
      for="three"
    >
      More than 3 years but less than 5
    </label>
  </div>
  <div
    className="radio"
  >
    <input
      id="four"
      name="experienceLevel"
      onChange={[Function]}
      type="radio"
      value="More than 5 years"
    />
    <label
      for="four"
    >
      More than 5 years
    </label>
  </div>
</div>
`;
